### PR TITLE
 runner-manager values file for gcloud

### PR DESCRIPTION
**Pull request by Abubakarr99**

### CODE REVIEW

File: charts/runner-manager/values-runner-manager-gcloud.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+githubToken: "vault:secret/data/runner-manager#GITHUB_TOKEN"
+redhatUsername: "vault:secret/data/runner-manager#REDHAT_USERNAME"
+redhatPassword: "vault:secret/data/runner-manager#REDHAT_PASSWORD"
+#redis:
+#  password: "vault:secret/data/runner-manager#rREDIS_PASSWORD"
+#allowedSshKeys: "vault:secret/data/runner-manager#allowed_ssh_keys"
+githubOrganization: scalanga-devl
+deployment:
+  ingress:
+    #enabled: true
```
Comment (gaspardmoindrot) [inline charts/runner-manager/values-runner-manager-gcloud.yaml:11]: Why did you comment on the whole igress? If it's done on purpose to test the service without and then implement it, I'm fine with it

File: charts/runner-manager/values-runner-manager-gcloud.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  minutes: 20
+  hours: 0
+pythonConfigModule: 'srcs.settings.settings_local'
+serviceMonitor:
+  enabled: true
+  labels:
+    monitoring-instance: prometheus-central
+
+image:
+  tag: '0.4.0'
```
Comment (gaspardmoindrot) [inline charts/runner-manager/values-runner-manager-gcloud.yaml:34]: You should make another release if you push new files

File: charts/runner-manager/values-runner-manager-gcloud.yaml (hunk 1 above)
Comment (Abubakarr99) [reply charts/runner-manager/values-runner-manager-gcloud.yaml:11]: yes exactly to test the service

File: charts/runner-manager/values-runner-manager-gcloud.yaml (hunk 2 above)
Comment (Abubakarr99) [reply charts/runner-manager/values-runner-manager-gcloud.yaml:34]: Ok

**Comment by Abubakarr99** [comment]
It seems we won't do this, so I am closing the PR

**Pull request closed without merging**